File: 2st/Yeonwoo/점의 위치 구하기.py
def solution(dot):
    answer = 0
    if dot[0] > 0 and dot[1] > 0:
        answer = 1
    elif dot[0] < 0 and dot[1] > 0:
        answer = 2
    elif dot[0] < 0 and dot[1] < 0:
        answer = 3
    elif dot[0] > 0 and dot[1] < 0:
        answer = 4
    return answer

File: 4st/Seohyun/programmer/점의 위치 구하기.java
class Solution {
    public int solution(int[] dot) {
        int answer = 0;
        if (dot[0]>0) {
            if (dot[1]>0)
                answer=1;
            else
                answer=4;
        }
        else if (dot[0]<0) {
            if (dot[1]>0)
                answer=2;
            else
                answer=3;
        }
        return answer;
    }
}